lsp-completion.el

```diff
@@ -715,6 +706,12 @@ The CLEANUP-FN will be called to cleanup."
   "Disable LSP completion support."
   (lsp-completion-mode -1))
 
+(defun lsp-completion-pass-through-try-completion (string table pred _point)
+  (try-completion string table pred))
```

Comment:
Using `try-completion` is not correct here. The function must follow the calling convention of `completion-try-completion` which (confusingly) is different from `try-completion`.